erdpy_network/proxy_network_provider.py

```diff
@@ -75,10 +75,10 @@ def get_nonfungible_token_of_account(self, address: IAddress, collection: str, n
         return token
 
     def get_transaction(self, tx_hash: str) -> TransactionOnNetwork:
-        """NOT IMPLEMENTED. Will raise NotImplementedError"""
-        url = self.__build_url_with_query_parameters(f'transaction/{tx_hash}', {'with_results': 'true'})
-        response = self.do_get_generic(url)
-        transaction = TransactionOnNetwork.from_proxy_http_response(tx_hash, response.to_dictionary())
+        url = self.__build_url_with_query_parameters(f'http://transaction/{tx_hash}', {'withResults': 'true'})
```

Comment:
`http://` seems off here.

Reply:
```
__build_url_with_query_parameters
```

Perhaps we can also use the simple approach, like in api network provider, and remove this function (especially if it has protocol / prefix requirements)?

Reply:
Solved